### ISSUE TITLE
Problem with video files with moving face or talking

### ISSUE THREAD

**Issue by ShmuelRonen** [kijai/ComfyUI-MimicMotionWrapper#13]
Thank you for the intriguing project!

In the video samples you put in the samples folder, the face hardly moves. Just changing expressions.
But when I select a video with speech, the result comes out distorted and jumpy. Is there a chance this will be resolved?

Example:

https://github.com/kijai/ComfyUI-MimicMotionWrapper/assets/80190186/e0b42943-9178-4aab-a891-97d26c8356bd

**Comment by mrkennypowers** [comment]
A new project called "Live Portrait" would probably be better suited for this specific task.

**Comment by ShmuelRonen** [comment]
Yes I put the problem in the wrong repository... But the great changes you made in Live Portrait are probably worth it!
